ndoh_hub/settings.py

```diff
@@ -350,7 +350,12 @@
 HCS_STUDY_C_REGISTRATION_FLOW_ID = env.str("HCS_STUDY_C_REGISTRATION_FLOW_ID", None)
 
 # ADA
+ADA_SYMPTOMCHECK_V2 = env.bool("ADA_SYMPTOMCHECK_V2", False)
 ADA_RAPIDPRO_URL = env.str("ADA_RAPIDPRO_URL", None)
 ADA_RAPIDPRO_TOKEN = env.str("ADA_RAPIDPRO_TOKEN", None)
 
 ADA_PROTOTYPE_SURVEY_FLOW_ID = env.str("ADA_PROTOTYPE_SURVEY_FLOW_ID", None)
+
+if ADA_SYMPTOMCHECK_V2:
+    RAPIDPRO_URL = env.str("RAPIDPRO_URL")
```

Comment:
are these different to the rapidpro settings on line 354?

and is it necessary to have the `ADA_SYMPTOMCHECK_V2` flag?